### DISCUSSION TITLE
Normal hotspot has no internet

### DISCUSSION THREAD

**Discussion by xNotta** [Mygod/VPNHotspot#210] [Q&A]
My normal (non-vpn) hotspot no longer has internet. The only way for me to use the hotspot is with the app. I have to enable the hotspot then enable wlan1.

If I don't use the app to start both, my hotspot is connectable but doesn't have internet.

It seems like I must be connected to my VPN with wlan1 enabled to use the hotspot for internet.

How can I fix it so I can return to using the built-in hotspot again?

(Android 11)

**Comment by Mygod** [comment]
Does it work after a restart?

**Comment by xNotta** [comment]
No.
The system hotspot doesn't work after a clear data, uninstall, or reinstall of VPNHotspot.

**Comment by Mygod** [comment]
Is it from your carrier's restrictions?

**Comment by xNotta** [comment]
I don't think it's a carrier restriction. The hotspot worked before I started using the app. 

Is there a way to check if my carrier recently found a way to block hotspot?

I read that the app doesn't modify anything on 10+ with safe mode. So if you've never heard of this happening and can't think of anything to try, then it's possible my hotspot got messed up some other way. 

I'm way past due for a factory reset anyways.

**Comment by Mygod** [comment]
If you have been using safe mode since the beginning, it is most likely that a phone restart should reset everything. If you have a second phone, you can try to plug your SIM into the other phone and see if it works. Anyway, you can also use this app to have normal hotspot by disabling normal tethering interface.

**Comment by knight0615** [comment]
I have the same question，but my wifi is normal after restarting my phone.
Everytime i use vpnhotspot app，i need to restart to make wifi normal.

**Reply by knight0615** [reply]
i want to know other way expect restart.
